Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -903,7 +903,7 @@
     accentVColor: "#447987"
 
 - type: entity
-  parent: BasePDA
+  parent: BaseMedicalPDA #Delta-v Make PyschologistPDA able to use medical scan
```

Comment:
because this is an upstream item; this could lead to a lot more future changes for the Psychologist PDA than just being a med scanner; which would be undocumented & maybe unwanted. It'd probably be safer to mirror the 'medscanner' component and tag it as Delta-V.

Reply:
i.e (indentation is dumb on github but)
```
- type: HealthAnalyzer
        scanDelay: 1

Reply:
I get the issue in theory, but I really can't think of anything else that would be in BaseMedicalPDA besides the medical scanner
I do think making the analyzer take longer is a good idea, so I will mirror anyways